portugues_pt/mundo/startup/startup.md

```diff
@@ -1 +1,6 @@
-Começava uma nova empresa com o novo produto revolucionário do mercado
\ No newline at end of file
+Começava uma nova empresa com o novo produto revolucionário do mercado. 
+A ideia era revolucionária, mas não foi a hora certa para fazê-la! Poucas pessoas
+compraram seu novo produto e sua startup começou a entrar numa grande crise!
+
+[Teria que demitir 70% dos funcionários para tentar se recuperar] (demitir/demitir.md)
+[Fazer acordo com funcionários para receberam 10% do salário durante um período] (acordo/acordo.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.